ci/README.md

```diff
@@ -1,9 +1,36 @@
-# CodeBuild / CodePipeline Testing
+# Using these scripts to deploy the auth0-custom-lock
+
+## Building the container
+
+You won't necessarily need to build the container.  But if you do you can build it with:
+
+`docker build . -t mozillaiam/auth0-custom-lock-builder:latest`
+
+## Assume the admin role
+
+This assumes you have access to infosec-dev and infosec-prod for CDN and S3 access using regular boto credential handlers.
+
+## Development
+
+Deploying using this from your local machine.
+```
+docker run -ti -v ~/.aws:/root/.aws -v `pwd`:/nlx -e CDN_BUCKET_NAME=sso-dashboard.configuration -e NODE_ENV=development -e CLOUDFRONT_DIST_ID=E3B9GI6602TZBY mozillaiam/auth0-custom-lock-builder:latest /bin/bash
```

Comment:
imo a makefile with these as target would be neat
Ex:
https://github.com/mozilla-iam/mozilla.oidc.accessproxy/blob/master/Makefile